### ISSUE TITLE
Override field size

### ISSUE THREAD

**Issue by mervynlee94** [umts/prawn-rails-forms#38]
Any idea how to override the method below to increase the field size in my rails application?

```
 def make_field_box(field)
    stroke_bounds
    bounds.add_left_padding 2
    move_down 2
    text field.upcase, size: 14
  end
```

I tried this in config/initializers/prawn_rails.rb but it is not working as expected.
```
include PrawnRailsForms

DocumentExtensions.module_eval do
  private
  def make_field_box(field)
    stroke_bounds
    bounds.add_left_padding 2
    move_down 2
    text field.upcase, size: 14
  end
end


```

**Comment by Anbranin** [comment]
This worked for me, in this initializer (underneath everything already there, leaving that untouched):
``` ruby
include PrawnRailsForms::DocumentExtensions
def make_field_box(field)
# your code with a custom size
end
```
Let me know if that works and I'll close out this issue.

**Comment by werebus** [comment]
Can you clarify what you mean by, "increase the field size"? It looks like what you're trying to do here is increase the font size of the label, this text:

![2021-11-23_09-06](https://user-images.githubusercontent.com/240275/143038943-4a7eb2c2-da8d-4390-b670-924635d6001b.png)

There are built-in ways to increase the row height (via the `height:` option to the `field_row` method), and the width, height, and font size (`size:`) of the `text_field`.

---
That said, monkey-patching our module _does_ appear to work:

```ruby
  PrawnRailsForms::DocumentExtensions.module_eval do
    def make_field_box(field)
      stroke_bounds
      bounds.add_left_padding 2
      move_down 2
      text field.upcase, size: 14
    end
  end
```

![2021-11-23_09-45](https://user-images.githubusercontent.com/240275/143045727-36f1e567-a6ee-4213-b806-e6e22c1add12.png)

Just be advised:

1. It's not guaranteed to keep working, it _is_ a private method, after all.
2. Doing so can break all sorts of other parts of your PDF. In our example app, I see several instances of missing field values because they got bushed out of the bounding box, checkboxes floating outside of their box, etc.

**Comment by mervynlee94** [comment]
> This worked for me, in this initializer (underneath everything already there, leaving that untouched):
> 
> ```ruby
> include PrawnRailsForms::DocumentExtensions
> def make_field_box(field)
> # your code with a custom size
> end
> ```
> 
> Let me know if that works and I'll close out this issue.

I actually copied the whole DocumentExtensions to make it work, starting with
```
include PrawnRailsForms

module PrawnRailsForms
  module DocumentExtensions
    ...
  end
end
```

**Comment by mervynlee94** [comment]
> Can you clarify what you mean by, "increase the field size"? It looks like what you're trying to do here is increase the font size of the label, this text:
> 
> ![2021-11-23_09-06](https://user-images.githubusercontent.com/240275/143038943-4a7eb2c2-da8d-4390-b670-924635d6001b.png)
> 
> There are built-in ways to increase the row height (via the `height:` option to the `field_row` method), and the width, height, and font size (`size:`) of the `text_field`.
> 
> That said, monkey-patching our module _does_ appear to work:
> 
> ```ruby
>   PrawnRailsForms::DocumentExtensions.module_eval do
>     def make_field_box(field)
>       stroke_bounds
>       bounds.add_left_padding 2
>       move_down 2
>       text field.upcase, size: 14
>     end
>   end
> ```
> 
> ![2021-11-23_09-45](https://user-images.githubusercontent.com/240275/143045727-36f1e567-a6ee-4213-b806-e6e22c1add12.png)
> 
> Just be advised:
> 
> 1. It's not guaranteed to keep working, it _is_ a private method, after all.
> 2. Doing so can break all sorts of other parts of your PDF. In our example app, I see several instances of missing field values because they got bushed out of the bounding box, checkboxes floating outside of their box, etc.

Yes you are right. The font size (`size:`) of the `text_field` only works on the `value` but not `field`. As I read from the source code, the `field` is hardcoded. I think it will be nice to include the customisation of `value` font size in the future.
